package.json

```diff
@@ -45,6 +46,7 @@
     "esprima": "4.0.0",
     "jshint": "2.9.5",
     "lebab": "2.7.7",
+    "lodash-es": "^4.17.5",
```

Comment:
Same here. Do we even need the explicit `lodash-es` dependency?

Reply:
Would committing the entire package to the vendor folder be a better place?